spring-cloud-services-spring-connector/src/test/java/io/pivotal/spring/cloud/service/config/PropertyMaskingContextInitializerIntegrationTests.java

```diff
@@ -81,11 +86,25 @@ public void vaultPropertyIsIncludedInSantizeEndpoints() {
 					.withFailMessage("Sanitize endpoints property not equal to " + sanitizeEndpointsProp.getValue())
 					.contains(VAULT_TEST_SANITIZE_PROPERTY);
 		}
+		
+		@Test
```

Comment:
Could we use `@ParameterizedTest` here to pass in the property name, instead of duplicating the test?

Reply:
We could, except this is connectors and connectors is still JUnit 4. So no `@ParameterizedTest`.

Reply:
Ah, then it's all good!